src/vivarium_gates_nutrition_optimization_child/data/builder.py

```diff
@@ -72,6 +75,8 @@ def load_and_write_data(
     else:
         logger.debug(f"Loading data for {key} for location {location}.")
         data = loader.get_data(key, location, fetch_subnationals)
+        if isinstance(data, pd.DataFrame) and ("location" in data.index.names):
+            data = scrub_location_level(data)
```

Comment:
Although this function name is consistent with the other "scrubbing" functions, I think it'd be more clear if it was called something like create_subnational_level. Maybe that's misleading as well though, so it's up to you